elements/lisk-cryptography/test/tsconfig.json

```diff
@@ -4,7 +4,15 @@
 		"baseUrl": ".",
 		"declaration": false,
 		"target": "es2017",
-		"types": ["node", "mocha", "chai", "globals", "buffer-reverse", "sodium-native", "varuint-bitcoin"],
+		"types": [
```

Comment:
probably we can remove this file

Reply:
We need this for types which are only used in tests.